analyzers/its/sources/Net5/Net5/S2292.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Net5
+{
+    public class S2292
```

Comment:
There's no S2292 in the modified JSON files

Reply:
See my comment above in the PR. This rule is still not merged and it's you who reviews it :)

Reply:
It's merged now, so we'll need a rebase and the JSON